### ISSUE TITLE
AttributeError: 'Headers' object has no attribute 'get'

### ISSUE THREAD

**Issue by naoko** [encode/apistar#332]
`Python 3.5.2`
`apistar==0.3.9`

I'm getting `AttributeError: 'Headers' object has no attribute 'get'` from `apistar/components/wsgi.py` in get_header.

This works on a few environments but it error out on others. I compare `pip freeze` output and all environment are identical.
On working instances, python versions are 3.5.3 or 3.5.4. 

and when I change the line from `.get` to `.__getitem__` all works.
```
def get_header(name: ParamName, headers: http.Headers):
    return headers.__getitem__(name.replace('_', '-'))
```
Any advice would be appreciated. 

UPDATE:
I've try to run on other system that has python 3.5.2 and got the same error. Now I'm convinced that this is issue on 3.5.2. Can we test with `hasattr`

**Comment by ratulotron** [comment]
Faced the same issue. Fixed by upgrading to Python 3.6.

**Comment by anseljh** [comment]
I ran into this one too, and it was rather sneaky. I'd been developing on 3.6.4, and then deployed to a server with 3.5.2. Instant breakage, even though `pip freeze` output was identical.

Relevant portion of my traceback:
```
  File "/home/username/.local/share/virtualenvs/api-gjVyAMqI/lib/python3.5/site-packages/apistar/frameworks/wsgi.py", line 134, in __call__
    response = self.http_injector.run_all(funcs, state=state)
  File "/home/username/.local/share/virtualenvs/api-gjVyAMqI/lib/python3.5/site-packages/apistar/components/dependency.py", line 141, in run_all
    ret = step.func(**kwargs)
  File "/home/username/.local/share/virtualenvs/api-gjVyAMqI/lib/python3.5/site-packages/apistar/frameworks/wsgi.py", line 130, in __call__
    response = self.http_injector.run_all(funcs, state=state)
  File "/home/username/.local/share/virtualenvs/api-gjVyAMqI/lib/python3.5/site-packages/apistar/components/dependency.py", line 141, in run_all
    ret = step.func(**kwargs)
  File "/home/username/.local/share/virtualenvs/api-gjVyAMqI/lib/python3.5/site-packages/apistar/components/wsgi.py", line 67, in get_header
    return headers.get(name.replace('_', '-'))
AttributeError: 'Headers' object has no attribute 'get'
```

Would love to see #364 get merged. 👏

**Comment by tomchristie** [comment]
Thanks for raising this... resolved in #400.
